cli/compilers/mod.rs

```diff
@@ -0,0 +1,15 @@
+use deno::ErrBox;
+use futures::Future;
+
+pub mod js;
+pub mod json;
+pub mod ts;
```

Comment:
Consider doing this instead:
```
pub use js::JsCompiler;
pub use json::JsonCompiler;
pub use ts::TsCompiler;
```
Then we don't expose these submodules to the caller